core/src/Kokkos_View.hpp

```diff
@@ -494,17 +494,11 @@ constexpr bool is_assignable(const Kokkos::View<ViewTDst...>& dst,
 
 namespace Kokkos {
 
-namespace {
+inline constexpr Kokkos::ALL_t ALL{};
 
-constexpr Kokkos::Impl::ALL_t ALL = Kokkos::Impl::ALL_t();
+inline constexpr Kokkos::Impl::WithoutInitializing_t WithoutInitializing{};
```

Comment:
`inline` specifier is not strictly necessary.  It is implicit since C++17.  Let me know if you want me to drop it.

Reply:
are you sure about c++17 implicitly inlining this? If i read here: https://en.cppreference.com/w/cpp/language/inline
i see "A static member variable (but **not a namespace-scope variable**) declared constexpr is implicitly an inline variable."

Reply:
Looks like I was wrong.  Thanks for pointing that out

Reply:
I did not know that either, just learned it too ..